doc/nrf/protocols/thread/overview/ot_memory.rst

```diff
@@ -67,19 +67,19 @@ The following tables present memory requirements for samples running on the :ref
    +-----------------------------+-------+-------+
    |                             |   FTD |   MTD |
    +=============================+=======+=======+
-   | ROM OT stack + App [kB]     |   327 |   274 |
+   | ROM OT stack + App [kB]     |   330 |   275 |
    +-----------------------------+-------+-------+
-   | ROM Bluetooth LE stack [kB] |    29 |    30 |
+   | ROM Bluetooth LE stack [kB] |    30 |    30 |
    +-----------------------------+-------+-------+
-   | Persistent storage [kB]     |    24 |    24 |
+   | Persistent storage [kB]     |    32 |    32 |
    +-----------------------------+-------+-------+
-   | Free ROM [kB]               |   644 |   696 |
+   | Free ROM [kB]               |   632 |   687 |
    +-----------------------------+-------+-------+
    | RAM OT stack + App [kB]     |   100 |    90 |
    +-----------------------------+-------+-------+
-   | RAM Bluetooth LE stack [kB] |    11 |    11 |
+   | RAM Bluetooth LE stack [kB] |    15 |    15 |
```

Comment:
Out of curiosity, why BLE stack size grows for nRF5340 and decreases for nRF52840?

Reply:
Noticed that as well, but verified the values and script works properly, so seems like so. I'll ask BLE team about that